README.md

```
@@ -10,315 +10,363 @@
+ - Inside the iex (you might need to hit enter/return to see the prompt)
+   - Update HSM templates by running the following command:
+   - `Glific.Templates.sync_hsms_from_bsp(1)`
+
+ Now you can visit [`https://glific.test:4001`](https://glific.test:4001) from your browser.
+
+## 6. Installed frontend
+  [Glific Frontend Repo](https://github.com/glific/glific-frontend)
+
+### Frontend credentials
+
+ * Phone: `917834811114`
```

markdownlint-cli2 (0.17.2)

Unordered list indentation
Expected: 0; Actual: 1

(MD007, ul-indent)